### PR TITLE
typo in updater message

### PR DESCRIPTION
**Pull request by a092devs**

### CODE REVIEW

File: userge/plugins/tools/updater.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ async def check_update(message: Message):
     await message.edit(
         f'`Now pushing updates from [{branch}] to heroku...\n'
         'this will take upto 3 min`\n\n'
-        '* **Restart** me after about 3 min using `.restart -h`\n\n'
+        '* **Restart** me after about 3 min using `{tr}restart -h`\n\n'
```
Comment (rking32) [inline userge/plugins/tools/updater.py:96]: this won't work
you have to use `{Config.CMD_TRIGGER}` instead of `{tr}`

Comment (FLAMEPOSEIDON) [reply userge/plugins/tools/updater.py:96]: Ok sur

Comment (a092devs) [reply userge/plugins/tools/updater.py]: 👍

**Comment by Phyco-Ninja** [comment]
You forgot f in start 👀

File: userge/plugins/tools/updater.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ async def check_update(message: Message):
     await message.edit(
         f'`Now pushing updates from [{branch}] to heroku...\n'
         'this will take upto 3 min`\n\n'
-        '* **Restart** me after about 3 min using `{tr}restart -h`\n\n'
+        '* **Restart** me after about 3 min using `{Config.CMD_TRIGGER}restart -h`\n\n'
```
Comment (rking32) [inline userge/plugins/tools/updater.py:96]: add f also like
```suggestion
        f'* **Restart** me after about 3 min using `{Config.CMD_TRIGGER}restart -h`\n\n'
```

**Pull request merged**